aten/src/ATen/native/layer_norm.cpp

```diff
@@ -0,0 +1,148 @@
+#include <ATen/NativeFunctions.h>
```

Comment:
Please move LayerNormForwardCPUImpl and all CPU logic into the /cpu subfolder using DECLARE_DISPATCH logic (see CopyKernel for the reference). It will allow to utilize AVX2 instructions and other optimizations of OSS build.

Reply:
Done